src/win32/kerberos_win32.cc

```diff
@@ -86,9 +90,14 @@ void KerberosClient::WrapData(const CallbackInfo& info) {
     std::string user = ToStringWithNonStringAsEmpty(options["user"]);
     int protect = 0; // NOTE: this should be an option
 
+    if (isStringTooLong(user)) {
+        throw Error::New(info.Env(), "User name is too long");
+        return;
```

Comment:
```suggestion
```

here and elsewhere, right? Otherwise this might end up generating compiler warnings about dead code.